.changelog.yml

```diff
@@ -1,4 +1,14 @@
+# The full repository name
```

Comment:
Not should be in another PR ❓ 🤔

Reply:
dont think this should be a problem - since this will get front-ported

Reply:
Just asking to understand behavior of working on gitea

Reply:
.changelog is config for the changelog-tool